### ISSUE TITLE
Model Bringup: Bert

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-forge-fe#431]
# Requirements

| Framework | Input source | Task                    | variant                                             | Device Type | Completed | Blocker |
|-----------|--------------|-------------------------|-----------------------------------------------------|-------------|-----------|---------|
| pytorch   | Transformers | Question answering      | bert-large-cased-whole-word-masking-finetuned-squad | N150        |    :white_check_mark:     |     https://github.com/tenstorrent/tt-mlir/issues/1449    |
| pytorch   | Transformers | Token Classification    | dbmdz/bert-large-cased-finetuned-conll03-english    | N150        |   :white_check_mark:       |      https://github.com/tenstorrent/tt-mlir/issues/1449    |
| pytorch   | Transformers | Sequence Classification | textattack/bert-base-uncased-SST-2                  | N150        | ❌         |   https://github.com/tenstorrent/tt-mlir/issues/1063     |
| pytorch   | Transformers | Masked LM               | bert-base-uncased                                   | N150        | :white_check_mark:        |     https://github.com/tenstorrent/tt-mlir/issues/1449  |

Note: variants marked with :white_check_mark: are compiled successfully. Issues in runtime are added in Blocker fields

**Comment by kamalrajkannan78** [comment]
Duplicate of https://github.com/tenstorrent/tt-forge-fe/issues/1301
